tests/foreman/api/test_multiple_paths.py

```diff
@@ -151,7 +151,7 @@ def test_get_status_code(self, entity_cls):
         @Assert: HTTP 200 is returned with an ``application/json`` content-type
 
         """
-        logger.debug('test_get_status_code arg: {0}'.format(entity_cls))
+        logger.debug('test_get_status_code arg: %s', entity_cls)
```

Comment:
I've looked in to this, and it is possible to choose how string interpolation is done. But this ability was added in Python 3.2. See [here](https://docs.python.org/3/library/logging.html#logging.Formatter).